connector_magento/models/product/common.py

```diff
@@ -108,12 +108,10 @@ def recompute_magento_qty(self):
         informations for each product.
         """
         # group products by backend
-        backends = defaultdict(self.browse)
-        for product in self:
-            backends[product.backend_id] |= product
-
-        for backend, products in backends.iteritems():
-            self._recompute_magento_qty_backend(backend, products)
+        for backend in self.mapped('backend_id'):
+            self._recompute_magento_qty_backend(
+                backend, self.filtered(
+                    lambda x: x.backend_id.id == backend.id))
```

Comment:
Ok, good to know that updating a recordset is so slow...it's  What about the version I propose?